src/components/DonationForm/DonationForm.jsx

```diff
@@ -132,39 +125,26 @@ function DonationForm() {
 
         <div className={styles['field-section']}>
           <h1 className={styles.title}>Furniture Submissions</h1>
-          {fields.map((furniture, index) => {
+          {furnitureFields.map((furniture, index) => {
             return (
               // eslint-disable-next-line react/no-array-index-key
               <div key={index}>
-                <h1 className={styles.title}> Furniture {index + 1} </h1>
-                <div className={styles['field-spacing']}>
-                  <FormLabel>Items to be Donated</FormLabel>
-                  <Select
-                    defaultChecked="Dressers"
-                    {...register(`furnitureField.${index}.itemName`)}
-                  >
-                    {furnitureOptions.map((furnitureItem, i) => {
-                      // eslint-disable-next-line react/no-array-index-key
-                      return <option key={i}>{furnitureItem}</option>;
-                    })}
-                  </Select>
-                </div>
-                <div className={styles['field-spacing']}>
-                  <FormLabel>Furniture Image Link</FormLabel>
-                  <Input defaultValue="" {...register(`furnitureField.${index}.imageLink`)} />
-                </div>
-                <div className={styles['field-spacing']}>
-                  <FormLabel>Description</FormLabel>
-                  <Input defaultValue="" {...register(`furnitureField.${index}.description`)} />
-                </div>
-                <Button onClick={() => remove(index)}>Delete</Button>
+                <FurnitureField
+                  index={index}
+                  register={register}
+                  removeFurniture={removeFurniture}
+                />
               </div>
```

Comment:
You don't need to wrap the FurnitureField component in a div. Also, if your arrow function only returns one item then you can drop the curly braces and return keyword